mlflow/ml-package-versions.yml

```diff
@@ -702,6 +704,7 @@ langchain:
 llama_index:
   package_info:
     pip_release: "llama-index"
+    module_name: "llama_index.core"
```

Comment:
note: This new is introduced to handle cases where the flavor name is different from the module name that is used for getting the library version.